pkg/arvo/app/spider.hoon

```diff
@@ -193,6 +204,23 @@
         tid.old
         ~
       ==
+    ::
+    ++  old-to-3
+      |=  old=clean-slate-any
+      ^-  (quip card clean-slate)
+      ?>  ?=(?(%2 %3) -.old)
+      ?:  ?=(%3 -.old)
+        `old
+      :-  ~
+      :*  %3
+        starting.old
+        running.old
+        tid.old
+        %-  ~(gas by *(map tid [@ta =mark =desk]))
+        %+  turn  ~(tap by serving.old)
+        |=([=tid id=@ta =mark] [tid id mark q.byk.bowl])
```

Comment:
We should remember to migrate HTTP bindings for threads to their new desks during the dist OTA.

Reply:
We don't necessarily need to do this, as all threads called via HTTP resolve in a single event.